### ISSUE TITLE
Could not pass event EntityDeathEvent to DropHeads v3.7.2.2

### ISSUE THREAD

**Issue by FastJoe0** [EvModder/DropHeads#64]
The error appears when someone joined the server.

```
[03:02:38] [Server thread/ERROR]: Could not pass event EntityDeathEvent to DropHeads v3.7.2.2
java.lang.NullPointerException: Cannot invoke "net.evmodder.DropHeads.listeners.DeathMessagePacketIntercepter.unblockDeathMessage(org.bukkit.entity.Entity)" because the return value of "net.evmodder.DropHeads.listeners.EntityDeathListener.access$1(net.evmodder.DropHeads.listeners.EntityDeathListener)" is null
	at net.evmodder.DropHeads.listeners.EntityDeathListener$DeathEventExecutor.execute(EntityDeathListener.java:286) ~[DropHeads-3.7.2.2.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:git-Purpur-1623]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:671) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R2.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:887) ~[purpur-1.18.2.jar:git-Purpur-1623]
	at net.minecraft.world.entity.LivingEntity.dropAllDeathLoot(LivingEntity.java:1781) ~[?:?]
	at net.minecraft.world.entity.LivingEntity.die(LivingEntity.java:1677) ~[?:?]
	at net.minecraft.world.entity.LivingEntity.hurt(LivingEntity.java:1492) ~[?:?]
	at net.minecraft.world.entity.LivingEntity.causeFallDamage(LivingEntity.java:1966) ~[?:?]
	at net.minecraft.world.level.block.Block.fallOn(Block.java:514) ~[purpur-1.18.2.jar:git-Purpur-1623]
	at net.minecraft.world.entity.Entity.checkFallDamage(Entity.java:1560) ~[purpur-1.18.2.jar:git-Purpur-1623]
	at net.minecraft.world.entity.LivingEntity.checkFallDamage(LivingEntity.java:375) ~[?:?]
	at net.minecraft.world.entity.Entity.move(Entity.java:1037) ~[purpur-1.18.2.jar:git-Purpur-1623]
	at net.minecraft.world.entity.LivingEntity.handleRelativeFrictionAndCalculateMovement(LivingEntity.java:2898) ~[?:?]
	at net.minecraft.world.entity.LivingEntity.travel(LivingEntity.java:2853) ~[?:?]
	at net.minecraft.world.entity.LivingEntity.aiStep(LivingEntity.java:3399) ~[?:?]
	at net.minecraft.world.entity.Mob.aiStep(Mob.java:659) ~[?:?]
	at net.minecraft.world.entity.monster.Monster.aiStep(Monster.java:44) ~[?:?]
	at net.minecraft.world.entity.monster.AbstractSkeleton.aiStep(AbstractSkeleton.java:111) ~[?:?]
	at net.minecraft.world.entity.LivingEntity.tick(LivingEntity.java:3001) ~[?:?]
	at net.minecraft.world.entity.Mob.tick(Mob.java:421) ~[?:?]
	at net.minecraft.world.entity.monster.Skeleton.tick(Skeleton.java:97) ~[?:?]
	at net.minecraft.server.level.ServerLevel.tickNonPassenger(ServerLevel.java:1218) ~[?:?]
	at net.minecraft.server.level.ServerLevel.lambda$tick$5(ServerLevel.java:697) ~[?:?]
	at net.minecraft.world.level.entity.EntityTickList.forEach(EntityTickList.java:43) ~[purpur-1.18.2.jar:git-Purpur-1623]
	at net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:675) ~[?:?]
	at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1619) ~[purpur-1.18.2.jar:git-Purpur-1623]
	at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:522) ~[purpur-1.18.2.jar:git-Purpur-1623]
	at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1482) ~[purpur-1.18.2.jar:git-Purpur-1623]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1246) ~[purpur-1.18.2.jar:git-Purpur-1623]
	at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:320) ~[purpur-1.18.2.jar:git-Purpur-1623]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

**Comment by EvModder** [comment]
Thanks for reporting it; it is fixed in [DropHeads v3.7.2.3](https://dev.bukkit.org/projects/dropheads/files/3797712) (will be available for download on Bukkit sortly)

**Comment by EvModder** [comment]
(It was happening when replace-behead-messages was disabled for both players and mobs, the plugin was trying to un- cancel a death message that never got cancelled in the first place lol)
